src/functions/template-tags/general.php

```diff
@@ -799,3 +799,18 @@ function tribe_asset_print_group( $group, $echo = true ) {
 		return $assets->print_group($group, $echo);
 	}
 }
+
+if ( ! function_exists( 'tribe_doing_shortcode' ) ) {
+	/**
+	 * Check if it's the doing the tribe_shortcode.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $tag
+	 *
+	 * @return bool If it's doing the shortcode
+	 */
+	function tribe_doing_shortcode( $tag = null ) {
+		return tribe( 'shortcode.manager' )->is_doing_shortcode( $tag );
```

Comment:
Should we just use the class name here so it's still using the same singleton instance?